truss/templates/trtllm/packages/build_engine_utils.py

```diff
@@ -0,0 +1,35 @@
+from pathlib import Path
+
+from schema import EngineBuildArgs
+
+
+def build_engine_from_config_args(
+    engine_build_args: EngineBuildArgs,
+    dst: Path,
+):
+    import sys
+
+    sys.path.append("/app/baseten")
+
+    import os
+    import shutil
+
+    # NOTE: These are provided by the underlying base image
```

Comment:
Can you pull `sys.path.append("/app/baseten")` down here to make it also explained by this note (espcially since the hardcoded path literal is assuming a very particular execution context).
And maybe make it actually a TODO, since I think we should long term find a more principled solution for this.